xclim/core/indicator.py

```diff
@@ -1107,6 +1037,10 @@ def datacheck(**das):  # noqa
             datachecks.check_freq(da, "H")
 
 
+base_registry["Hourly"] = Hourly
+base_registry["Daily"] = Daily
```

Comment:
Because of the need of `Indicator.from_dict` to be able to parse the  `base` string, we need to have a registry of base classes that are incomplete indicators (i.e. are not in the main registry).